sequential/seq2pat.py

```diff
@@ -123,6 +125,9 @@ class _Constants:
     # List where index correspond to the attribute ids and values to the minimum value for that attribute
     min_attrs = 'min_attrs'
 
+    # Minimum row count as a threshold of mined patterns
```

Comment:
hm.. is this a new parametr? If not, how come we didn't have it before? are we adding a new constraint or sth

Reply:
This is still not used though. I added it here when I make a change to`_get_cython_imp()`, to use the seq2pat backend to implement the encoding, but I didn't choose that route then and this should be removed.